README.md

```diff
@@ -56,7 +56,9 @@ if (;false) {
 
 ## Declarations
 
-There are four types of declaration. Constant constants can't be changed in any way.
+There are four types of declaration. 
+
+1. Constant constants can't be **changed** in any way.
```

Comment:
if you add a `1.` here, you should continue the enumeration on the other entries